models/ArticleManager.php

```diff
@@ -0,0 +1,369 @@
+<?php
+
+class ArticleManager extends Model
+{
+// FONCTIONS POUR LE FRONTEND
+    // Fonction qui permet de récupérer les détails des articles postés
+    public function get_posts_blog(){
+        $req = $this->db->query("
+            SELECT  articles.id,
+                    articles.title,
+                    articles.chapo,
+                    articles.image,
+                    articles.date,
+                    articles.content,
+                    admins.name
+            FROM articles
+            JOIN admins
+            ON articles.writer=admins.email
+            WHERE posted='1'
+            ORDER BY date DESC
+            LIMIT 0,5
+        ");
+        
+        while($row = $req->fetch()){
+            
+            $post   = new Article();
+            $post   ->setId($row['id']);
+            $post   ->setTitle($row['title']);
+            $post   ->setChapo($row['chapo']);
+            $post   ->setContent($row['content']);
+            $post   ->setDate($row['date']);
+            $post   ->setImage($row['image']);
+                        
+            $posts[] = $post;
+        }
+        return $posts;
+    }
+    
+     public function get_posts_blog_all(){
+        $req = $this->db->query("
+            SELECT  articles.id,
+                    articles.title,
+                    articles.chapo,
+                    articles.image,
+                    articles.date,
+                    articles.content,
+                    admins.name
+            FROM articles
+            JOIN admins
+            ON articles.writer=admins.email
+            WHERE posted='1'
+            ORDER BY date DESC
+        ");
+        
+        while($row = $req->fetch()){
+            
+            $post   = new Article();
+            $post   ->setId($row['id']);
+            $post   ->setTitle($row['title']);
+            $post   ->setChapo($row['chapo']);
+            $post   ->setContent($row['content']);
+            $post   ->setDate($row['date']);
+            $post   ->setImage($row['image']);
+                        
+            $posts[] = $post;
+        }
+        return $posts;
+    }
+    
+    // Fonction qui récupère un article avec le posted = 1
+    public function get_article_blog(){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [get_article_blog accesses the super-global variable $_GET.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)